tools/scripts/mysql_migration.py

```
@@ -1893,19 +1839,15 @@ def execute(self) -> tuple[int, list]:
 
                 # Get rows where tenant_*_id is NULL or empty
                 cursor = self.db.execute_sql(
-                    f"SELECT id, `{legacy_col}` FROM `{table_name}` "
-                    f"WHERE (`{tenant_id_col}` IS NULL OR `{tenant_id_col}` = '') "
-                    f"AND `{legacy_col}` IS NOT NULL AND `{legacy_col}` != ''"
+                    f"SELECT id, `{legacy_col}` FROM `{table_name}` WHERE (`{tenant_id_col}` IS NULL OR `{tenant_id_col}` = '') AND `{legacy_col}` IS NOT NULL AND `{legacy_col}` != ''"
```

Ruff (0.15.20)

[error] Possible SQL injection vector through string-based query construction

(S608)